hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/HapiTestEnv.java

```diff
@@ -90,18 +91,13 @@ public HapiTestEnv(@NonNull final String testName, final boolean cluster, final
     /**
      * Starts all nodes in the environment.
      */
-    public void start() {
+    public void start() throws TimeoutException {
         started = true;
         for (final var node : nodes) {
             node.start();
         }
-
         for (final var node : nodes) {
-            try {
-                node.waitForActive(CAPTIVE_NODE_STARTUP_TIME_LIMIT);
-            } catch (TimeoutException e) {
-                throw new RuntimeException(e);
-            }
```

Comment:
Why the removal of this catch?

Reply:
It isn't necessary and made an _already handled_ exception into an unhandled runtime exception.